src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/documents/recallnotification/LetterToProbationHtmlGenerationTest.kt

```diff
@@ -18,31 +21,50 @@ class LetterToProbationHtmlGenerationTest(
   private val underTest = LetterToProbationGenerator(templateEngine)
 
   @Test
-  fun `generate in custody letter to probation HTML`(approver: ContentApprover) {
+  fun `generate in custody fixed recall letter to probation HTML`(approver: ContentApprover) {
     approver.assertApproved(
       underTest.generateHtml(
-        letterToProbationContext(true),
+        letterToProbationContext(true, FIXED),
       )
     )
   }
 
   @Test
-  fun `generate not in custody letter to probation HTML`(approver: ContentApprover) {
+  fun `generate not in custody fixed recall letter to probation HTML`(approver: ContentApprover) {
     approver.assertApproved(
       underTest.generateHtml(
-        letterToProbationContext(false),
+        letterToProbationContext(false, FIXED),
       )
     )
   }
 
-  private fun letterToProbationContext(inCustody: Boolean) = LetterToProbationContext(
+  @Test
+  fun `generate in custody standard recall letter to probation HTML`(approver: ContentApprover) {
+    approver.assertApproved(
+      underTest.generateHtml(
+        letterToProbationContext(true, STANDARD),
+      )
+    )
+  }
+
+  @Test
+  fun `generate not in custody standard recall letter to probation HTML`(approver: ContentApprover) {
+    approver.assertApproved(
+      underTest.generateHtml(
+        letterToProbationContext(false, STANDARD),
+      )
+    )
+  }
+
+  private fun letterToProbationContext(inCustody: Boolean, recallType: RecallType) = LetterToProbationContext(
     LocalDate.of(2021, 9, 29),
     RecallLengthDescription(FOURTEEN_DAYS),
     "Mr probation",
     FullName("Bertie Offender"),
     "bookingNumber",
-    PrisonName("Current prison name"),
+    if (!inCustody && recallType == FIXED) null else PrisonName("Current prison name"),
```

Comment:
I've maybe missed something but why `&& recallType == FIXED` here?

Reply:
Current Prison Name is not used when NIC on a Fixed Term Recall. For all other cases we use it.

Reply:
Hmm, just checking the logic...

Reply:
Good spot 🤦‍♂️